### PR TITLE
WSTEAM1-1024 Add Uploader Form links to our existing Lighthouse tests

### PR DESCRIPTION
**Pull request by emilysaffron**
Resolves JIRA [WSTEAM1-1024](https://jira.dev.bbc.co.uk/browse/WSTEAM1-1024)

Overall changes
======
Adds links to the uploader form to our lighthouse tests

Code changes
======

- _A bullet point list of key code changes that have been made._

Testing
======
1. _List the steps used to test this PR._

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

### CODE REVIEW

File: ws-nextjs-app/pages/[service]/send/[id]/[[...variant]].page.tsx [hunk 1]
```diff
@@ -60,7 +60,7 @@ export const getServerSideProps: GetServerSideProps = async context => {
           }
         : null,
       pageType: UGC_PAGE,
-      pathname: null,
+      pathname: context.resolvedUrl,
       service,
```
Comment (emilysaffron) [inline ws-nextjs-app/pages/[service]/send/[id]/[[...variant]].page.tsx:71]: There are lots of things in these props that we're not setting, when we are setting them for the live pages. Is this intentional? Do we need a ticket written to fill these in ?

Comment (amoore108) [reply ws-nextjs-app/pages/[service]/send/[id]/[[...variant]].page.tsx:71]: We should probably have them more or less match between the 2.

Things like `ads` probably aren't needed for the uploader though, but the rest will be useful to have.

Comment (amoore108) [reply ws-nextjs-app/pages/[service]/send/[id]/[[...variant]].page.tsx:71]: If its simple to add now that would be great. But if its a lot more work, a dedicated ticket would be better.

File: scripts/lighthouseRun.sh [hunk 2]
```diff
@@ -4,3 +4,5 @@ npx -p lighthouse@11.3.0 lighthouse http://localhost:7080/news/articles/cn7k01xp
...
 npx -p lighthouse@11.3.0 lighthouse http://localhost:7081/pidgin/live/c7p765ynk9qt?renderer_env=test --chrome-flags="--no-sandbox --headless --disable-gpu" --output json --output html --output-path simorgh --config-path scripts/lighthouseConfig.js && node scripts/lighthouseBudget.js run
 npx -p lighthouse@11.3.0 lighthouse http://localhost:7081/pidgin/live/c07zr0zwjnnt?renderer_env=test --chrome-flags="--no-sandbox --headless --disable-gpu" --output json --output html --output-path simorgh --config-path scripts/lighthouseConfig.js && node scripts/lighthouseBudget.js run
+npx -p lighthouse@11.3.0 lighthouse http://localhost:7081/mundo/send/u50853489?renderer_env=live --chrome-flags="--no-sandbox --headless --disable-gpu" --output json --output html --output-path simorgh --config-path scripts/lighthouseConfig.js && node scripts/lighthouseBudget.js run
+npx -p lighthouse@11.3.0 lighthouse http://localhost:7081/somali/send/u130092370?renderer_env=test --chrome-flags="--no-sandbox --headless --disable-gpu" --output json --output html --output-path simorgh --config-path scripts/lighthouseConfig.js && node scripts/lighthouseBudget.js run
```
Comment (karinathomasbbc) [inline scripts/lighthouseRun.sh:8]: What's the difference between the 2 uploader forms here (and do we need to test both of them?)

**Review by karinathomasbbc — APPROVED** [review]

File: ws-nextjs-app/pages/[service]/send/[id]/[[...variant]].page.tsx (hunk 1 above)
Comment (emilysaffron) [reply ws-nextjs-app/pages/[service]/send/[id]/[[...variant]].page.tsx:71]: okay, ill make a seperate ticket for it as i dont think it falls under this 👍

File: scripts/lighthouseRun.sh (hunk 2 above)
Comment (emilysaffron) [reply scripts/lighthouseRun.sh:8]: the somali one allows for files to be uploaded, and the mundo is just comments - think we will need to check both to ensure a11y is passing mainly for both functionalities 😊

**Review by amoore108 — APPROVED** [review]

File: scripts/lighthouseRun.sh (hunk 2 above)
Comment (karinathomasbbc) [reply scripts/lighthouseRun.sh:8]: Thanks for explaining, makes sense

File: ws-nextjs-app/pages/[service]/send/[id]/[[...variant]].page.tsx (hunk 1 above)
Comment (emilysaffron) [reply ws-nextjs-app/pages/[service]/send/[id]/[[...variant]].page.tsx:71]: actually thinking about it i think its simple enough to add as part of this, sorry hadnt refreshed and seen your comment !

Comment (amoore108) [reply ws-nextjs-app/pages/[service]/send/[id]/[[...variant]].page.tsx:71]: Awesome, thanks for this!

**Review by amoore108 — APPROVED** [review]

**Pull request merged**